docs/release_process.rst

```diff
@@ -0,0 +1,175 @@
+    vi RELEASE.rst # paste results of prior command at top of file, correcting the release number
+    git add RELEASE.rst lore/settings.py
+    git commit -m "Release 0.10.0"
+    # verify that the Travis-ci build succeeds
+    # Wait for all checkboxes to be checked
+    # Check with Dev-Ops buddy whether migrations need to be run for this release
+    git flow release publish 0.10.0
+    git push --tags
+    # Send release notification to LORE mailing list - odl-lore-announce@mit.edu
+
+Annotated Instructions
+----------------------
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+
+Update the ``VERSION`` number in ``lore/settings.py`` following Semantic
+numbering practice.
+
+.. code-block:: bash
+
+    vi lore/settings.py # change version number
+
+Create the release notes with a utility called ``git-release-notes``.
+https://www.npmjs.com/package/git-release-notes  Its output is written
+to the console.
+
+``git-release-notes`` takes two command line arguments.  The first argument
+takes a range of git references to determine which commits to include in
+the release.  In the example given it is all commits from the last release
+to what is in ``master``.
+
+The second command argument specifies a template to format the output.
+LORE has two templates.
+
+- ``util/release_notes_rst.ejs`` formats the output as
+  reStructuredText for inclusion in ``RELEASE.rst``.
+
+- ``util/release_notes.ejs`` formats the output as to precede each commit
+  with a checkbox, and follow each with a link to the developer's commit
+  history.  Add these release notes as the description for the GitHub PR
+  to the ``release`` branch.
+
+Generate the release notes in reStructuredText format:
+
+.. code-block:: bash
+
+  git-release-notes v0.9.0..master util/release_notes_rst.ejs
+
+Update ``release.rst`` with this version's release notes (the console output
+generated by running ``release_notes_rst.ejs``).  Edit the new
+release notes to correct the release number, make each statement in the
+past tense, capitalize the first word, and end sentences with a period.
+
+.. code-block:: bash
+
+    vi RELEASE.rst
+
+Add, commit, and push the two files you have updated to publish
+the release.
+
+.. code-block:: bash
+
+    git add RELEASE.rst lore/settings.py
+    git commit -m "Release 0.10.0"
+    git flow release publish 0.10.0
+
+The commit will trigger a build on Travis CI.
+https://travis-ci.org/mitodl/lore/builds  For each commit throughout
+the release, it's good practice to check that the build succeeded
+before proceeding.
+
+Run ``git-release-notes`` again to create the Pull Request descriptions for
+the ``release`` branch.
+
+.. code-block:: bash
+
+  git-release-notes v0.9.0..master util/release_notes.ejs
+
+and paste the Markdown output as the description of the PR to the release
+branch ``rc/0.10.0`` (This is the one with the checkboxes.)
```

Comment:
Do you want to move these lines (106-107) after the creation of the PRs (112-115)?